Unix/pal/palcommon.h

```diff
@@ -22,7 +22,7 @@
 #    include <windows.h>
 #    include <strsafe.h>
 #else
-#    include "../../common/linux/sal.h"
+#    include "linux/sal.h"
```

Comment:
Seeing `<linux/sal.h>` makes me think "system include file". If this is a local include file, I prefer `"linux/sal.h"` or something like that.